<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewContact</class>
 <widget class="QMainWindow" name="NewContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>691</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Новый контакт</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddContact">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавить контакт</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="fname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите пол:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="man">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Мужской</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="woman">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Женский</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Имя *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Телефон *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QComboBox" name="countryphone">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Россия, +7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Беларусь, +375</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Украина, +380</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>США, +1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Англия, +44</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Китай, +86</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Индия, +91</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
